src/bootstrap/doc.rs

```diff
@@ -375,7 +375,7 @@ impl Step for Standalone {
                up_to_date(&footer, &html) &&
                up_to_date(&favicon, &html) &&
                up_to_date(&full_toc, &html) &&
-               up_to_date(&version_info, &html) &&
+               (builder.config.dry_run || up_to_date(&version_info, &html)) &&
                (builder.config.dry_run || up_to_date(&rustdoc, &html)) {
```

Comment:
```suggestion
               builder.config.dry_run ||
               (up_to_date(&version_info, &html) && up_to_date(&rustdoc, &html)) {
```

Reply:
This hurts rather than aids readability in my eyes.